src/Accelerators/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+
+if( ACCELERATORS_TO_BUILD )
+  foreach(t ${ACCELERATORS_TO_BUILD})
+    message(STATUS "Targeting ${t}")
+    add_subdirectory(${t})
+    endforeach(t)
+endif( ACCELERATORS_TO_BUILD )
+
+
+
+
+  add_onnx_mlir_library(NoAccel
```

Comment:
If you  believe it can be removed later, just add a "todo, remove"

Reply:
let me think about it